### PR TITLE
Change $JULIA_DIR to $JULIA_PREFIX

### PR DESCRIPTION
**Pull request by spacewander**
According to the talk in https://groups.google.com/forum/#!topic/julia-users/vXC5Z_aTKjM

### CODE REVIEW

**Comment by stevengj** [comment]
LGTM.

File: doc/manual/embedding.rst [hunk 1]
```diff
@@ -33,9 +33,10 @@ We start with a simple C program that initializes Julia and calls some Julia cod
       return 0;
   }
 
-In order to build this program you have to put the path to the Julia header into the include path and link against ``libjulia``. For instance, when Julia is installed to ``$JULIA_DIR``, one can compile the above test program ``test.c`` with gcc using::
+In order to build this program you have to put the path to the Julia header into the include path and link against ``libjulia``. For instance, when Julia is installed to ``$JULIA_PREFIX/bin``, one can compile the above test program ``test.c`` with gcc using::
 
-    gcc -o test -I$JULIA_DIR/include/julia -L$JULIA_DIR/usr/lib -ljulia test.c
+    # $JULIA_PREFIX/lib is the directory where libjulia lies in
```
Comment (stevengj) [inline doc/manual/embedding.rst]: This comment looks a bit odd, and also doesn't explain  the `-I` flag.  Maybe remove the comment and change the text to:

```
For instance, when Julia is installed to ``$JULIA_PREFIX/bin`` (where you are responsible for defining the ``JULIA_PREFIX`` variable appropriately) , its header file ``julia.h`` is installed in ``$JULIA_PREFIX/include`` and its library ``libjulia`` is installed in ``$JULIA_PREFIX/lib`` (by default). In this case, one can compile the above ``test.c`` program with gcc using::

**Comment by spacewander** [comment]
:) Ok, I will update it.

**Comment by ViralBShah** [comment]
Is the build failure relevant? If not, perhaps we can merge.

File: doc/manual/embedding.rst [hunk 2]
```diff
@@ -33,9 +33,9 @@ We start with a simple C program that initializes Julia and calls some Julia cod
       return 0;
   }
 
-In order to build this program you have to put the path to the Julia header into the include path and link against ``libjulia``. For instance, when Julia is installed to ``$JULIA_DIR``, one can compile the above test program ``test.c`` with gcc using::
+In order to build this program you have to put the path to the Julia header into the include path and link against ``libjulia``. For instance, when Julia is installed to ``$JULIA_PREFIX/bin`` (where you are responsible for defining the ``JULIA_PREFIX`` variable appropriately) , its header file ``julia.h`` is installed in ``$JULIA_PREFIX/include`` and its library ``libjulia`` is installed in ``$JULIA_PREFIX/lib`` (by default). In this case, one can compile the above ``test.c`` program with gcc using::
```
Comment (stevengj) [inline doc/manual/embedding.rst]: you have an extra space before a `,`.   There should also be a comma after "In order to build this program".

While I am nitpicking, probably the paragraph should be consistent about "you" vs. "one" — pick one and use it consistently.

**Comment by ararslan** [comment]
No longer relevant, as this environment variable has since been renamed.

**Pull request closed without merging**